third_party/nanopb

```diff
@@ -1 +1 @@
-Subproject commit f8ac463766281625ad710900479130c7fcb4d63b
```

Comment:
Yeah, something's broken with Jenkins at the moment, and isn't reporting anything. I was suspicious, and ran the tests myself.